README.md

```diff
@@ -33,21 +33,3 @@ are installed in /usr/local/bin.  See the INSTALL file for more information.
 Run `man percona-toolkit` to see a list of installed tools, then `man tool`
 to read the embedded documentation for a specific tool.  You can also read
 the documentation online at [http://www.percona.com/software/percona-toolkit/](http://www.percona.com/software/percona-toolkit/).
-
-## Version 3
```

Comment:
why you delete this section?

Reply:
:thinking:

Reply:
Because it is not relevant anymore. I moved it into CONTRIBUTE.md first, then figured out that all this "put code into GOPATH" staff is not required. Go programs work as is.